### ISSUE TITLE
Generator Adversarial Loss

### ISSUE THREAD

**Issue by dmerrick520** [automan000/SimGAN_PyTorch#1]
I believe the following line of code for the adversarial loss for the generator is not right. This is line 221 in main.py ...

`adv_loss = self.local_adversarial_loss(refined_predictions, refined_labels)`               

Shouldn't it be the following line of code since the generator should be generating images that trick the discriminator into thinking the refined images are real?

`adv_loss = self.local_adversarial_loss(refined_predictions, real_labels)`

**Comment by koutilya40192** [comment]
Hi @dmerrick520. Were you able to get better realistic refined images in comparison with the results posted by @automan000 by modifying the adv_loss equation?

**Comment by dmerrick520** [comment]
Hi @koutilya40192 ,

I was able to get decent results. Check out my repository with results here, https://github.com/dmerrick520/Learning-from-Simulated-and-Unsupervised-Images-through-Adversarial-Training-SimGAN-PyTorch .. It is largely based on @automan000 code but with some changes.

**Comment by koutilya40192** [comment]
Thank you so much for sharing @dmerrick520 . The results definitely look more pleasing. It seems to me that your version of simgan was able to generate realistic eyes. Just wondering about the gaze part though. Was it being unchanged during this refinement process? Were you able to reproduce the gaze estimation results from the original paper?

**Comment by dmerrick520** [comment]
@koutilya40192 I was able to get a couple degree improvement from training on refined eyes, however I was not able to reach the accuracy in the paper. I haven't played around with hyper parameters too much, though. Also, I'm not certain if I'm using the same accuracy metrics, etc because I don't believe the authors go into much detail about that in the paper.
